src/main/java/org/fenixedu/academic/ui/struts/action/commons/student/enrollment/bolonha/AbstractBolonhaStudentEnrollmentDA.java

```diff
@@ -191,6 +194,10 @@ public ActionForward enrolInOptionalCurricularCourse(ActionMapping mapping, Acti
             request.setAttribute("optionalEnrolmentBean", optionalStudentEnrollmentBean);
 
             return mapping.findForward("chooseOptionalCurricularCourseToEnrol");
+        } catch (org.fenixedu.bennu.core.domain.exceptions.DomainException de) {
```

Comment:
The other place in this class where a DomainException is used could be replaced by this type